bin/cron/delete_accounts

```diff
@@ -5,13 +5,15 @@ require 'cdo/delete_accounts_helper'
 require 'cdo/only_one'
 
 def main
+  delete_accounts_helper = DeleteAccountsHelper.create
```

Comment:
Ah, for some reason I didn't see this the first time around and thought it was a typo here. 

The static helper is fine, but I do think it would be a bit more idiomatic and discoverable if the constructor had default arguments. Your call.